src/aliceVision/voctree/CMakeLists.txt

```diff
@@ -25,7 +25,6 @@ alicevision_add_library(aliceVision_voctree
     aliceVision_feature
     aliceVision_sfmData
     aliceVision_system
-    Eigen3::Eigen
```

Comment:
this should stay because of `DefaultAllocator` that explicitly uses `Eigen::aligned_allocator`